src/coreclr/vm/interpreter.cpp

```diff
@@ -9297,6 +9299,21 @@ void Interpreter::DoCallWork(bool virtualCall, void* thisArg, CORINFO_RESOLVED_T
                 DoGetIsSupported();
                 didIntrinsic = true;
             }
+
+            if (strcmp(methodName, "get_IsHardwareAccelerated") == 0 && strcmp(namespaceName, "System.Runtime.Intrinsics") == 0)
+            {
+                GCX_COOP();
+                DoGetIsSupported();
+                didIntrinsic = true;
+            }
+
+            if (strcmp(methodName, "get_IsSupported") == 0 && strcmp(className, "EventSource") == 0 && strcmp(namespaceName, "System.Diagnostics.Tracing") == 0)
```

Comment:
If EventSource does not work, it would be better to disable for now by passing `-p:System.Diagnostics.Tracing.EventSource.IsSupported=false` on your command line instead of adding workarounds here.